### DISCUSSION TITLE
Add support for composer-dependency-analyser

### DISCUSSION THREAD

**Discussion by janedbal** [shivammathur/setup-php#858] [Ideas / RFC]
I'd like to add support for [composer-dependency-analyser](https://github.com/shipmonk-rnd/composer-dependency-analyser?tab=readme-ov-file#composer-dependency-analyser) tool: [PR](https://github.com/shivammathur/setup-php/pull/859)

**Comment by shivammathur** [comment]
We have a issue for this open https://github.com/shivammathur/setup-php/issues/841. So closing this.
